ocs_ci/ocs/pgsql.py

```diff
@@ -287,7 +287,10 @@ def wait_for_pgbench_status(self, status, timeout=None):
             timeout (int): Time in seconds to wait
 
         """
-        timeout = timeout if timeout else 900
+        if status == constants.STATUS_RUNNING:
+            timeout = 900
+        elif status == constants.STATUS_COMPLETED:
+            timeout = 9000
```

Comment:
You dont need to have an additional condition for the timeout, you could directly pass the time out value to the function